### PR TITLE
Added test for invalid command line arguments

### PR DESCRIPTION
**Pull request by jGleitz**

### CODE REVIEW

File: src/final1/subtests/InvalidCommandLineArgumentsTest.java [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+    @Test
+    public void noArgumentTest() {
+        exitTest("");
+    }
+
+    /**
+     * Asserts that the tested class prints an error message if more than one command line argument is provided.
+     */
+    @Test
+    public void toManyArgumentsTest() {
```
Comment (pcworld) [inline src/final1/subtests/InvalidCommandLineArgumentsTest.java]: *too

**Comment by pcworld** [comment]
What happens if one tries to open an existent folder as a file?
Might be worth a test, because this probably results in different JDK behavior.
Btw, we currently use `SystemExitStatus.WITH_GREATER_THAN_0` everywhere, even though the task sheet states exit code should be `1`.
Otherwise, +1 for merge.

**Comment by jGleitz** [comment]
See #54

**Comment by jGleitz** [comment]
@pcworld I thought about testing against binary files and folders, too. But while we could create a folder to test with at runtime, I don't know how to create a binary file (=> thus unreadable for `FileReader`) with java.

**Comment by pcworld** [comment]
> unreadable for FileReader

Are you sure anything is unreadable for FileReader? If anything, binary files could lead to wrong behavior in a user's program.

**Comment by jGleitz** [comment]
> Are you sure anything is unreadable for FileReader?

I assumed that bytes that don't form a code point in a encoding lead to a exception. I have not tried it, though.

**Comment by jGleitz** [comment]
Anyway, we should merge this.

**Comment by pcworld** [comment]
yup

**Pull request merged**